src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double sideLength;
+
+    public Square(String color, double sideLength) {
+        super(color);
+        this.sideLength = sideLength;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method name `area` should follow the Java naming conventions for methods, which typically use verbs. Consider renaming the method to something like `calculateArea` to clearly indicate the action it performs.